src/Microsoft.AspNet.WebHooks.Custom.SqlStorage/WebHooks/WebHookStoreContext.cs

```diff
@@ -13,15 +16,58 @@ namespace Microsoft.AspNet.WebHooks
     public class WebHookStoreContext : DbContext
     {
         internal const string ConnectionStringName = "MS_SqlStoreConnectionString";
-        private const string ConnectionStringNameParameter = "name=" + ConnectionStringName;
+        private readonly string _tableName;
+        private readonly string _schemaName = "WebHooks";
 
         /// <summary>
         /// Initializes a new instance of the <see cref="WebHookStoreContext"/> class.
         /// </summary>
-        public WebHookStoreContext() : base(ConnectionStringNameParameter)
+        public WebHookStoreContext() : base("name=" + ConnectionStringName)
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="WebHookStoreContext"/> class using the given string
+        /// as the name or connection string for the database to which a connection will be made.
+        /// </summary>
+        /// <param name="nameOrConnectionString">Either the database name or a connection string.</param>
+        public WebHookStoreContext(string nameOrConnectionString) : base(nameOrConnectionString)
+        {
+            NameOrConnectionString = nameOrConnectionString;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="WebHookStoreContext"/> class using the given parameters
+        /// as the name or connection string for the database to which a connection will be made.
+        /// And configures the database schema name and table name.
+        /// </summary>
+        /// <param name="nameOrConnectionString">Either the database name or a connection string.</param>
+        /// <param name="schemaName">Either the schema name.</param>
+        /// <param name="tableName">Either the table name.</param>
+        public WebHookStoreContext(string nameOrConnectionString, string schemaName, string tableName)
+            : this(nameOrConnectionString)
+        {
+            if (string.IsNullOrEmpty(schemaName))
+            {
+                string msg = string.Format(CultureInfo.CurrentCulture, SqlStorageResources.SqlStore_EmptyString, nameof(schemaName));
```

Comment:
Use `var` as much as possible. This PR is generally pretty good about this but please scan from here to the end.